rholang/project/BNFC.scala

```diff
@@ -1,5 +1,6 @@
 import sbt._
 import Keys._
+import scala.sys.process._
```

Comment:
is this needed to get the code to compile with scala 2.12.4?

Reply:
Yep, I guess in sbt 1.x you need to explicitly import `process` package members to use.